doc/source/FAQ.rst

```diff
@@ -20,6 +20,23 @@ top-10 prediction for each user.
     :name: top_n_recommendations.py
     :lines: 10-
 
+.. _precision_recall_at_k:
+
+How to evaluate an algorithm using precision and recall at k metrics
+-----------------------------------------------------------------------
+
+Below is an example on how to compute Precision@k and Recall@k metrics for
+a recommendation model. We first split our dataset into train and test 
+folds. Using cross validation and on each iteration we train on our trainset 
+and get the predictions from the testset. Finally we compute Precision@k 
+and Recall@k for each user in predictions.
```

Comment:
"Finally we compute Precision@k and Recall@k for each user in predictions."

Could be expanded into:

-----------------------------------
Finally, we compute Precision@k and Recall@k for each user:
- Precision@k = #{Recommended items that are relevant} / #{recommended items}
- Recall@k = #{Recommended items that are relevant} / #{relevant items}

An item is considered relevant if its rating is greater than the threshold k. An item is considered recommended if its estimated rating is greater than the threshold k, and if it is among the k highest estimated ratings.

------------------------------
As the file will be proessed by sphinx, we could use latex formatting, e.g.

    precision@k = :math:`\frac{ | \{ \text{Recommended items that are relevant} \} | }{ | \{ \text{Recommended items} \} | }`
Or something like that. I'm not entirely sure about the syntax as I can't check it from github.